instrumentation/apache-camel-2.20/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/apachecamel/decorators/KafkaSpanDecorator.java

```diff
@@ -38,6 +39,11 @@
   private static final String TOPIC = "kafka.TOPIC";
   private static final String OFFSET = "kafka.OFFSET";
 
+  private static final boolean CAPTURE_EXPERIMENTAL_SPAN_ATTRIBUTES =
+      Config.get()
+          .getBooleanProperty(
+              "otel.instrumentation.apache-camel.experimental-span-attributes", false);
```

Comment:
Hm, can you use the constant from `BaseSpanDecorator` if you make it protected?

Reply:
👍 will make this change (in this PR) tomorrow